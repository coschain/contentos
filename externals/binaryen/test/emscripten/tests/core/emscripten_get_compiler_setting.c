#include <stdio.h>
#include <assert.h>
#include <emscripten.h>

int main() {
  printf("QS: %d\n", emscripten_get_compiler_setting("QUANTUM_SIZE"));
  assert((unsigned)emscripten_get_compiler_setting("OPT_LEVEL") <= 3);
  assert((unsigned)emscripten_get_compiler_setting("DEBUG_LEVEL") <= 4);
  printf("EV: %s\n", (char*)emscripten_get_compiler_setting("EMSCRIPTEN_VERSION"));
}

