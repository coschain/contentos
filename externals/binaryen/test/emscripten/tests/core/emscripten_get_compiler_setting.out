QS: 4
EV: waka
